src/lib/moment/prototype.js

```diff
@@ -147,4 +147,8 @@ proto.years  = deprecate('years accessor is deprecated. Use year instead', getSe
 proto.zone   = deprecate('moment().zone is deprecated, use moment().utcOffset instead. http://momentjs.com/guides/#/warnings/zone/', getSetZone);
 proto.isDSTShifted = deprecate('isDSTShifted is deprecated. See http://momentjs.com/guides/#/warnings/dst-shifted/ for more information', isDaylightSavingTimeShifted);
 
+proto[Symbol.for('nodejs.util.inspect.custom')] = function () {
```

Comment:
You need to check if Symbol is defined here. There is no code transpilation going on (only modularization), so this code should just work on old environments.